test/util/sqs.spec.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path');
+const expect = require('chai').expect;
+const nockBack = require('nock').back;
+const index = require('./../../src/index');
+const helper = require('../helper');
+
+describe('Testing sqs util', () => {
+  let aws;
+  before(() => {
+    helper.before();
+    aws = index();
+    nockBack.setMode('record');
+    nockBack.fixtures = path.join(__dirname, '..', '__cassettes', 'util', 'sqs');
+  });
+
+  after(() => helper.after());
+
+  describe('Testing sendMessageBatch', () => {
+    it('Testing send message success', (done) => {
+      nockBack('send-message-batch-success_recording.json', async (nockDone) => {
```

Comment:
Let's try to keep the recording naming consistent,
Rules are to keep the recording name and test name as close as possible and
`camelCaseName.json_recording.json`